Lesson3/README.md

```diff
@@ -60,12 +64,18 @@ Please follow the [Lesson 3: Configure and run the BLE sample application to rec
    ```
 10. **`gulp run --iot-hub`** - Run the BLE sample application and read IoT Hub messages that have just been sent from SensorTag.
 
-## Deploy and run simulate device sample
-1. Use the following command to modify your configuration, change the `"has_sensortag"` value from `true` to `false`
+## <a id="run-simudev"></a>Deploy and run simulate device cloud upload sample
+1. Use the following command to modify your configuration, change the `"has_sensortag"` value from `true` to `false`.
+
+   ```bash
+   code ../config.json
+   ```
 2. **`npm install`** - Install the npm packages
-3. **`gulp init`** - Intialize configuration files in user's profile folder `%USERPROFILE%\.iot-hub-getting-started`
+3. > *If you have run the script with `has_sensortag: true` flag before, you can run `gulp clean` to clean your personal config, this command will clean your edited config files in your profile folder created by `gulp init`, and tools installed on NUC by `gulp install-tools` command*
+

```

Comment:
*If you have run the script with `has_sensortag: true` before, you can run `gulp clean` to clean your environment. This command removes config files generated by `gulp init` in your profile folder as well as tools installed by `gulp install-tools` on the NUC*